### PR TITLE
Simplifying boolean returns

### PR DESCRIPTION
**Pull request by NathanCheshire**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.

### CODE REVIEW

**Comment by NathanCheshire** [comment]
Note to reviewers; after reading `contributing.md`, this type of "fix" didn't really fall into any of the bullet points listed. Please let me know if some type of further action is required on my end.

File: src/main/java/com/thealgorithms/maths/DudeneyNumber.java [hunk 1]
```diff
@@ -32,11 +32,6 @@ public static boolean isDudeney(int n) {
             temp /= 10;
         }
 
-        //If the cube root of the number is not equal to the sum of its digits we return false.
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/DudeneyNumber.java:35]: this comment line should be there. other than that everything looks good.

Comment (NathanCheshire) [reply src/main/java/com/thealgorithms/maths/DudeneyNumber.java:35]: Added back in even though it directly describes exactly what the code does. Is it not irrelevant anymore?

Comment (debasishbsws) [reply src/main/java/com/thealgorithms/maths/DudeneyNumber.java:35]: yup you are right.

**Review by debasishbsws — APPROVED** [review]

**Pull request merged**